packages/styles/scss/components/_richtext.scss

```diff
@@ -333,6 +331,10 @@
       );
     }
 
+    iframe {
```

Comment:
I don't think this is doing anything, there's no `figure` in this example that I can see.

Reply:
I got it confused here @justintemps since the initial issue was about adding spacing to iframe. Will make the change above and test it